pydatastructs/graphs/algorithms.py

```diff
@@ -877,3 +879,73 @@ def _job(graph: Graph, u: str):
     if len(L) != num_vertices:
         raise ValueError("Graph is not acyclic.")
     return L
+    
+def minDistance(graph: Graph,dist,visited,source):
+    min = sys.maxsize
```

Comment:
I think `float('inf')` is better alternative for `sys.maxsize`.

Reply:
```suggestion
    min = float('inf')
```